cannon/mipsevm/exec/mips_instructions32_test.go

```diff
@@ -108,3 +108,52 @@ func TestStoreSubWord_32bits(t *testing.T) {
 		})
 	}
 }
+
+func TestSignExtend_32bit(t *testing.T) {
```

Comment:
This function fails to test the cases that caused the SLL(V) sign extension issue. Namely, it doesn't test values with bits set on positions higher than index. In particular, it doesn't test sign extension with bits set above 32, when index is 32.

Reply:
Yes, probably a good idea to add additional coverage for higher index values in the 64-bit VM.  Added a follow-up issue [here](https://github.com/ethereum-optimism/optimism/issues/14318).